src/sql/src/plan/statement/ddl.rs

```diff
@@ -731,6 +731,7 @@ pub fn plan_create_source(
                 let mut nullable = true;
                 for option in &c.options {
                     match &option.option {
+                        ColumnOption::Null => (),
```

Comment:
Aah, that's what you meant! I didn't know there was an explicit `NULL` options, I thought you either write nothing or `NOT NULL`

Reply:
hah yes! I should have sent you the code